docs/src/components/productMaterial/MaterialStyling.tsx

```diff
@@ -151,7 +145,7 @@ export default function MaterialStyling() {
   }
   React.useEffect(() => {
     if (infoRef.current) {
-      infoRef.current.scroll({ top: scrollTo[index], behavior: 'smooth' });
+      infoRef.current.scroll({ top: scrollTo[index] * 0.75 * 16 * 1.5 - 3, behavior: 'smooth' });
```

Comment:
Where do these magic numbers come from?

Reply:
Great point. I have added comment and refactored the logic.